contracts/pcv/ERC20HoldingPCVDeposit.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.13;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {IPCVDeposit} from "./IPCVDeposit.sol";
+import {PCVDeposit} from "./PCVDeposit.sol";
+import {CoreRef} from "../refs/CoreRef.sol";
+import {TribeRoles} from "../core/TribeRoles.sol";
+import {Constants} from "../Constants.sol";
+import {IERC20HoldingPCVDeposit} from "./IERC20HoldingPCVDeposit.sol";
+import {MainnetAddresses} from "../test/integration/fixtures/MainnetAddresses.sol";
+import {ArbitrumAddresses} from "../test/integration/fixtures/ArbitrumAddresses.sol";
+
+/// @title ERC20HoldingPCVDeposit
+/// @notice PCVDeposit that is used to hold ERC20 tokens as a safe harbour. Deposit is a no-op
+contract ERC20HoldingPCVDeposit is PCVDeposit, IERC20HoldingPCVDeposit {
```

Comment:
add a comment up here to not use this contract until the WETH chainid has been sorted out.